src/Commands/MakeApiDocsCommand.php

```diff
@@ -0,0 +1,209 @@
+
+        $namespace = text(
+            label: 'In which namespace would you like to create this API Docs Resource in?',
+            default: $serverNameSpace
+        );
+
+        $handlersVirtualNamespace = "{$namespace}\\{$resourceClass}\\Handlers";
+        $transformersVirtualNamespace = "{$namespace}\\{$resourceClass}\\Transformers";
+
+        $baseResourceVirtualPath = (string) str($resourceClass)->prepend('/')->prepend($serverPath)->replace('\\', '/')->replace('//', '/');
+
+        $handlerVirtualDirectory = "{$baseResourceVirtualPath}/Handlers/";
+        $transformersVirtualDirectory = "{$baseResourceVirtualPath}/Transformers/";
+
+        if (method_exists($resource, 'getApiTransformer')) {
+            // generate API transformer
+            $transformer = ($modelNamespace . "\\" . $resourceClass)::getApiTransformer();
+            $transformerClassPath = (string) str($transformer);
+            $transformerClass = (string) str($transformerClassPath)->afterLast('\\');
+
+            $stubVars = [
+                'namespace' => $namespace,
+                'modelClass' => $pluralModelClass,
+                'resourceClass' => $resourceClass,
+                'transformerName'   => $transformerClass,
+            ];
+
+            if (!$this->checkForCollision(["{$transformersVirtualDirectory}/{$transformerClass}.php"])) {
+                $this->copyStubToApp("Api/Transformer", $transformersVirtualDirectory . '/' . $transformerClass . '.php', $stubVars);
+            }
+        }
+
+        try {
+            $handlerMethods = File::allFiles($handlerSourceDirectory);
+
+            foreach ($handlerMethods as $handler) {
+                $handlerName = basename($handler->getFileName(), '.php');
+                // stub exists?
+                if (!$this->fileExists($this->getDefaultStubPath() . "/ApiDocs{$handlerName}")) {
+
+                    if (!$this->checkForCollision(["{$handlerVirtualDirectory}/{$handlerName}.php"])) {
+
+                        $this->copyStubToApp("Api/{$handlerName}", $handlerVirtualDirectory . '/' . $handlerName . '.php', [
+                            'handlersVirtualNamespace' => $handlersVirtualNamespace,
+                            'transformersVirtualNamespace' => $transformersVirtualNamespace,
+                            'resource' => $resource,
+                            'resourceClass' => $resourceClass,
+                            'realResource' => "{$resource}\\Api\\Handlers\\{$handlerName}",
+                            'resourceNamespace' => $modelNamespace,
+                            'modelClass' => $modelClass,
+                            'pluralClass' => $pluralModelClass,
+                            'handlerClass' => $handler,
+                            'handlerName' => $handlerName,
+                            'capitalsResource' => strtoupper($modelClass),
+                            'path' => '/' . strtolower($pluralModelClass),
```

Comment:
please use `str($pluralModelClass)->kebab()` instead of strtolower, because we use this method to generate the url for PascalCase classname